src/main/java/jsr223/docker/compose/utils/DockerComposePropertyLoader.java

```diff
@@ -35,6 +35,7 @@
 package jsr223.docker.compose.utils;
 
 import lombok.AccessLevel;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'lombok.AccessLevel'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AUselessImportCheck)